src/librustc_typeck/check/mod.rs

```diff
@@ -5302,6 +5302,53 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
         Some(original_span.with_lo(original_span.hi() - BytePos(1)))
     }
 
+    // Rewrite `SelfCtor` to `StructCtor`
+    pub fn rewrite_self_ctor(&self, def : Def, span : Span) -> (Def, DefId, Ty<'tcx>) {
```

Comment:
Nit picking: the Rust style is `def: Def, span: Span`.

Reply:
Fixed.